### ISSUE TITLE
Latest update yeets creative menu

### ISSUE THREAD

**Issue by Crankonator** [CyberRat7/Extra_Compat#8]
```
[04May2024 23:19:01.734] [Render thread/ERROR] [net.minecraftforge.fml.javafmlmod.FMLModContainer/LOADING]: Caught exception during event net.minecraftforge.event.BuildCreativeModeTabContentsEvent@7a8e4d8f dispatch for modid extra_compat
java.lang.NullPointerException: Cannot invoke "net.cyber_rat.extra_compat.core.CompatCore.addCreative(net.minecraftforge.event.BuildCreativeModeTabContentsEvent)" because "compatCore" is null
	at net.cyber_rat.extra_compat.ExtraCompat.addCreative(ExtraCompat.java:29) ~[extra_compat-1.5.0.jar%23617!/:1.5.0]
	at net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:260) ~[eventbus-6.0.5.jar%2387!/:?]
	at net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:252) ~[eventbus-6.0.5.jar%2387!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.5.jar%2387!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.5.jar%2387!/:?]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:114) ~[javafmllanguage-1.20.1-47.2.30.jar%23832!/:?]
	at net.minecraftforge.fml.ModLoader.lambda$postEvent$29(ModLoader.java:326) ~[fmlcore-1.20.1-47.2.30.jar%23831!/:?]
	at java.lang.Iterable.forEach(Iterable.java:75) ~[?:?]
	at net.minecraftforge.fml.ModList.forEachModInOrder(ModList.java:227) ~[fmlcore-1.20.1-47.2.30.jar%23831!/:?]
	at net.minecraftforge.fml.ModLoader.postEvent(ModLoader.java:326) ~[fmlcore-1.20.1-47.2.30.jar%23831!/:?]
	at net.minecraftforge.common.ForgeHooks.onCreativeModeTabBuildContents(ForgeHooks.java:1633) ~[forge-1.20.1-47.2.30-universal.jar%23835!/:?]
	at net.minecraft.world.item.CreativeModeTab.m_269498_(CreativeModeTab.java:129) ~[client-1.20.1-20230612.114412-srg.jar%23830!/:?]
	at mezz.jei.library.plugins.vanilla.ingredients.ItemStackListFactory.create(ItemStackListFactory.java:61) ~[jei-1.20.1-forge-15.3.0.4.jar%23675!/:15.3.0.4]
	at mezz.jei.library.plugins.vanilla.VanillaPlugin.registerIngredients(VanillaPlugin.java:171) ~[jei-1.20.1-forge-15.3.0.4.jar%23675!/:15.3.0.4]
	at mezz.jei.library.load.PluginLoader.lambda$new$2(PluginLoader.java:78) ~[jei-1.20.1-forge-15.3.0.4.jar%23675!/:15.3.0.4]
	at mezz.jei.library.load.PluginCaller.redirect$chd000$emi$callOnPlugins(PluginCaller.java:541) ~[jei-1.20.1-forge-15.3.0.4.jar%23675!/:15.3.0.4]
	at mezz.jei.library.load.PluginCaller.callOnPlugins(PluginCaller.java:27) ~[jei-1.20.1-forge-15.3.0.4.jar%23675!/:15.3.0.4]
	at mezz.jei.library.load.PluginLoader.<init>(PluginLoader.java:78) ~[jei-1.20.1-forge-15.3.0.4.jar%23675!/:15.3.0.4]
	at mezz.jei.library.startup.JeiStarter.start(JeiStarter.java:113) ~[jei-1.20.1-forge-15.3.0.4.jar%23675!/:15.3.0.4]
	at mezz.jei.forge.startup.StartEventObserver.transitionState(StartEventObserver.java:137) ~[jei-1.20.1-forge-15.3.0.4.jar%23675!/:15.3.0.4]
	at mezz.jei.forge.startup.StartEventObserver.onEvent(StartEventObserver.java:100) ~[jei-1.20.1-forge-15.3.0.4.jar%23675!/:15.3.0.4]
	at net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:260) ~[eventbus-6.0.5.jar%2387!/:?]
	at net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:252) ~[eventbus-6.0.5.jar%2387!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.5.jar%2387!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.5.jar%2387!/:?]
	at net.minecraft.client.multiplayer.ClientPacketListener.m_5859_(ClientPacketListener.java:1451) ~[client-1.20.1-20230612.114412-srg.jar%23830!/:?]
	at net.minecraft.network.protocol.game.ClientboundUpdateTagsPacket.m_5797_(ClientboundUpdateTagsPacket.java:35) ~[client-1.20.1-20230612.114412-srg.jar%23830!/:?]
	at net.minecraft.network.protocol.game.ClientboundUpdateTagsPacket.m_5797_(ClientboundUpdateTagsPacket.java:11) ~[client-1.20.1-20230612.114412-srg.jar%23830!/:?]
	at net.minecraft.network.protocol.PacketUtils.m_263899_(PacketUtils.java:22) ~[client-1.20.1-20230612.114412-srg.jar%23830!/:?]
	at net.minecraft.util.thread.BlockableEventLoop.m_6367_(BlockableEventLoop.java:156) ~[client-1.20.1-20230612.114412-srg.jar%23830!/:?]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.m_6367_(ReentrantBlockableEventLoop.java:23) ~[client-1.20.1-20230612.114412-srg.jar%23830!/:?]
	at net.minecraft.util.thread.BlockableEventLoop.m_7245_(BlockableEventLoop.java:130) ~[client-1.20.1-20230612.114412-srg.jar%23830!/:?]
	at net.minecraft.util.thread.BlockableEventLoop.m_18699_(BlockableEventLoop.java:115) ~[client-1.20.1-20230612.114412-srg.jar%23830!/:?]
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1106) ~[client-1.20.1-20230612.114412-srg.jar%23830!/:?]
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:718) ~[client-1.20.1-20230612.114412-srg.jar%23830!/:?]
	at net.minecraft.client.main.Main.main(Main.java:218) ~[forge-47.2.30.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111) ~[fmlloader-1.20.1-47.2.30.jar:?]
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.clientService(CommonLaunchHandler.java:99) ~[fmlloader-1.20.1-47.2.30.jar:?]
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$makeService$0(CommonClientLaunchHandler.java:25) ~[fmlloader-1.20.1-47.2.30.jar:?]
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:108) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:78) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) ~[bootstraplauncher-1.1.2.jar:?]
	
```

**Comment by Crankonator** [comment]
[2024-05-04-3.log](https://github.com/CyberRat7/Extra_Compat/files/15212699/2024-05-04-3.log)

**Comment by Mi6kbuttface** [comment]
I got the same issue

**Comment by CyberRat7** [comment]
we are aware the newest version has many issues and are working on a fix please use an older version of the mod

**Comment by CyberRat7** [comment]
Fixed in 1.5.3 make sure you are using Temporal Api 1.6.6
